diff --git a/src/Hats.sol b/src/Hats.sol
index 37b156c..12029f1 100644
--- a/src/Hats.sol
+++ b/src/Hats.sol
@@ -16,7 +16,7 @@
 
 pragma solidity >=0.8.13;
 
-import { ERC1155 } from "lib/ERC1155/ERC1155.sol";
+import { ERC1155 } from "../lib/ERC1155/ERC1155.sol";
 // import { console2 } from "forge-std/Test.sol"; //remove after testing
 import "./Interfaces/IHats.sol";
 import "./HatsIdUtilities.sol";
